mscore/capella.cpp

```diff
@@ -337,7 +338,7 @@ static void processBasicDrawObj(QList<BasicDrawObj*> objects, Segment* s, int tr
                         QPointF p(st->pos());
                         p = p / 32.0 * score->spatium();
                         // text->setUserOff(st->pos());
-                        text->setAutoplace(false);
+                        text->setAutoplace(true);
```

Comment:
See the failures on Travis, https://travis-ci.org/github/musescore/MuseScore/jobs/671221637#L4406-L4452, due to this change. IMHO your change is good, so the reference files would need to get adjusted.

Seems that testText1.capx and testVolta1.capx are broken, Capella Reader refuses to open them:
[image](https://user-images.githubusercontent.com/1786669/78499635-1e718080-7752-11ea-83fc-252f4e43542b.png)
So it is difficult to tell how they should get rendered, but without autoplace the ref files don't look good

Reply:
Okay, so should I revert the change or update the ref files?

Reply:
the latter.

@lvinken what's up with those 2 capx files, my Capella Reader 8 can't open them (see above)

Reply:
IIRC the capx testfiles were never meant to be compatible with Capella: they were hand crafted to contain only the data actually used by MuseScore's Capella import. Most likely they do not even pass schema validation.

Reply:
I had that suspicion. Maybe I can one day twist a mate's arm and get him to generated those sample files with his Capella ;-)

Reply:
@lvinken That change to not disable autoplace though is fine by you?

Reply:
Fine by me (should have been done as early as 3.0 I guess ...)